src/make_data/banner_intervention/banner_selects_one_result.sql

```diff
@@ -0,0 +1,45 @@
+/* This query extracts the total number and proportion of sessions that selects
+   the checker via the banner intervention that also select at least one link on
+   the results page
+*/
+
+
+DECLARE start_date STRING DEFAULT "20210803";
+DECLARE end_date STRING DEFAULT "20210816";
+
+CREATE OR REPLACE TABLE `govuk-bigquery-analytics.banner_intervention.banner_one_result` AS
+
+-- All distinct sessions that select the checker via the banner (eventAction =
+-- interventionClicked)
+
+WITH
+  sessions_select_banner AS (
+    SELECT DISTINCT
+      CONCAT(fullVisitorId, "-", visitId) AS sessionId
+    FROM `govuk-bigquery-analytics.87773428.ga_sessions_*`
+    CROSS JOIN UNNEST(hits) AS hits
+    WHERE _TABLE_SUFFIX BETWEEN start_date AND end_date
```

Comment:
```suggestion
    WHERE _TABLE_SUFFIX BETWEEN start_date AND end_date
    AND hits.eventInfo.eventCategory = 'interventionBanner'
```
perhaps?

Reply:
I decided to leave this out as every `eventAction = interventionClicked` will have a corresponding `eventCategory = interventionBanner`, so this isn't really needed. 

Do you think it improves the readability and I should include?

Reply:
This would be good for extra caution, in case there are any events where `eventAction = interventionClicked` or Rejected where the `eventCategory` is not `interventionBanner` (i.e. someone uses the same eventAction text for a different event)

Reply:
I see you've noted this in the README so I'll resolve this comment :)